apis/comms_api/authentication/test/test_authentication.py

```diff
@@ -59,7 +60,7 @@ async def test_jwt_bearer_decode_ko(decode_token_mock):
     with pytest.raises(HTTPError) as exc:
         _ = await jwt(mock_req)
 
-    assert str(exc.value) == f'{status.HTTP_403_FORBIDDEN}: message'
+    assert str(exc.value) == '2706: Invalid authentication token'
```

Comment:
You can do this with the `match` argument of `pytest.raises`.

Something like:

```python
with pytest.raises(HTTPError, match='.* 2706.*') as exc:
        _ = await jwt(mock_req)
```